src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const paginationItemsCount = Math.ceil(total / perPage);
+  const next = currentPage >= paginationItemsCount;
+  const previous = currentPage === 1;
+
+  const handleNextLinkClick = () => {
+    if (currentPage !== paginationItemsCount) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  const handlePreviousLinkClick = () => {
+    if (currentPage !== 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames(['page-item', previous ? 'disabled' : ''])}
+        onClick={handlePreviousLinkClick}
```

Comment:
The 'onClick' event will still be fired even when the 'previous' link is disabled, which might not be the intended behavior. It would be better to conditionally handle clicks only when the previous page is available.